rust_book_ex_8_3/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+pub mod vector_ex;
+pub mod string_ex;
+pub mod hashmap_ex;
+
+#[cfg(test)]
+mod tests {
+    use super::vector_ex;
+    use super::string_ex;
+    use super::hashmap_ex;
+
+    #[test]
+    fn test_vector_exercise() {
+        assert_eq!(vector_ex::get_mean(vec![1, 2, 3, 4, 5]), 3.0);
```

Comment:
You could `use vector_ex::{get_mean, get_median, get_mode}` here, either inside the function or inside tests mod, so you don't have to repeat `vector_ex::` all the time.